07-generic_bookworms/final/Makefile

```diff
@@ -0,0 +1,14 @@
+run:
+	@go run .
+
+build:
+	@go build -o bin/gordle main.go
```

Comment:
Wrong binary name. Also, on windows it should be `bin/example.exe`. If you specify the directory instead you'll avoid both problems.

Similarly, use the current directory as the build rather than a specific filename, this is to avoid problems when people decide to split files out.

So, this can be:
```
@go build -o ./bin/ .
```

_I assume these need to be fixed in other chapters as well. But make a separate PR for them, not in this one._

Reply:
God catach. We'll update all of them. Thanks !